dataset/dictionary/characters.json5

```
@@ -5819,7 +5832,374 @@
     notes: "ナタを守る竜戦士。第五章第四幕に登場する人物",
     tags: [ "natlan", "character-sub" ],
   },
-
+  {
+    en: "Mutota",
+    ja: "ムトタ",
+    zhCN: "穆托塔",
+    notesEn: "Chief, Flower-Feather Clan",
+    notes: "「花翼の集」族長",
+    notesZh: "「花羽会」首领",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Yopia",
+    ja: "ヨピア",
+    zhCN: "尤庇娅",
+    notes: "第五章幕間に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Hueliyo",
+    ja: "ウエリヨ",
+    zhCN: "胡里约",
+    notes: "第五章幕間に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Matlan",
+    ja: "マトラン",
+    zhCN: "马特兰",
+    notes: "第五章幕間に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Atoco",
+    ja: "アトコ",
+    zhCN: "阿图科",
+    notes: "第五章幕間に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Cohitta",
+    ja: "コチッタ",
+    zhCN: "柯奇妲",
+    notes: "第五章幕間に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Ocotlan",
+    ja: "オコトラン",
+    zhCN: "奥科兰",
+    notes: "第五章幕間に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Izcalli",
+    ja: "イスカリ",
+    zhCN: "伊卡力",
+    notes: "第五章幕間に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Cuallee",
+    ja: "クアリー",
+    zhCN: "库阿勒",
+    notes: "第五章幕間に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Munay",
+    ja: "ムナイ",
+    zhCN: "穆纳伊",
+    notes: "第五章幕間に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Abundi",
+    ja: "アバンディ",
+    zhCN: "阿邦迪",
+    notesEn: "Obsidian Totem Pole manager of \"Flower-Feather Clan\"",
+    notes: "「花翼の集」の黒曜石トーテムポール管理人",
+    notesZh: "「花羽会」曜石图腾柱管理员",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Cusco",
+    ja: "クスコ",
+    zhCN: "库斯可",
+    notes: "チャスカの義父",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Allpa",
+    ja: "アルパ",
+    zhCN: "阿尔帕",
+    notes: "部族見聞「塵に帰す花、堕ちる羽」に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Coyllur",
+    ja: "コリユール",
+    zhCN: "科优尔",
+    notes: "部族見聞「塵に帰す花、堕ちる羽」に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Zunan",
+    ja: "ムッキ",
+    zhCN: "穆基",
+    notes: "部族見聞「塵に帰す花、堕ちる羽」に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Huascar",
+    ja: "ワスカール",
+    zhCN: "瓦斯卡尔",
+    notes: "部族見聞「塵に帰す花、堕ちる羽」に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Pixica",
+    ja: "ピシカ",
+    zhCN: "皮希加",
+    notes: "部族見聞「塵に帰す花、堕ちる羽」に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Coya",
+    ja: "コヤ",
+    zhCN: "柯娅",
+    notes: "部族見聞「塵に帰す花、堕ちる羽」に登場するチャスカの姉同然のクク竜",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Chime",
+    ja: "チメ",
+    zhCN: "奇梅",
+    notes: "部族見聞「塵に帰す花、堕ちる羽」に登場するコヤの子供のクク仔竜",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "himpu",
```

Should it be Capitalized since Himpu is a name?

```suggestion
    en: "Himpu",
```